### PR TITLE
Greedy algorithm

### PR DESCRIPTION
**Pull request by DanOgiela**
Please review changes

### CODE REVIEW

File: ecs/elections/models.py [hunk 1]
```diff
@@ -97,9 +97,11 @@ class Meta:
 
 BRUTE_ALGORITHM = 'b'
 GENETIC_ALGORITHM = 'g'
+GREEDY_ALGORITHM = 'r'
 ALGORITHM_CHOICES = (
     (BRUTE_ALGORITHM, 'Brute force'),
-    (GENETIC_ALGORITHM, 'Genetic')
+    (GENETIC_ALGORITHM, 'Genetic'),
+    (GREEDY_ALGORITHM, 'Greedy Algorithm')
```
Comment (jakubste) [inline ecs/elections/models.py]: ,

File: ecs/elections/views.py [hunk 2]
```diff
@@ -216,7 +217,8 @@ class ResultCreateView(ConfigureElectionMixin, CreateView):
     def form_valid(self, form):
         result = super(ResultCreateView, self).form_valid(form)
         algorithm = {
-            BRUTE_ALGORITHM: BruteForce
+            BRUTE_ALGORITHM: BruteForce,
+            GREEDY_ALGORITHM: GreedyAlgorithm
```
Comment (jakubste) [inline ecs/elections/views.py]: ,

File: ecs/elections/algorithms/greedy_algorithm.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
+from ecs.elections.algorithms.algorithm import Algorithm
+from ecs.utils.math import ell_p_norm
+
+
+class GreedyAlgorithm(Algorithm):
+    preferences = None
+
+    def fetch_preferences(self):
```
Comment (kasptom) [inline ecs/elections/algorithms/greedy_algorithm.py]: If the method body stays the same like in Algorithm class, we don't need to override it

File: ecs/elections/algorithms/greedy_cc.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
+from ecs.elections.algorithms.algorithm import Algorithm
+
+
+class GreedyCC(Algorithm):
+    preferences = None
+
+    def fetch_preferences(self):
```
Comment (kasptom) [inline ecs/elections/algorithms/greedy_cc.py]: Same like in GreedyAlgorithm

File: ecs/elections/algorithms/greedy_cc.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+from ecs.elections.algorithms.algorithm import Algorithm
+
+
+class GreedyCC(Algorithm):
```
Comment (kasptom) [inline ecs/elections/algorithms/greedy_cc.py]: We need to test this class

**Pull request merged**